ambry-replication/src/test/java/com/github/ambry/replication/LeaderBasedReplicationTest.java

```diff
@@ -0,0 +1,1177 @@
+package com.github.ambry.replication;
```

Comment:
@Arun-LinkedIn you need to add license here to pass the travis test. See other file for reference.

Reply:
Thanks for letting me know. Added the license text.